tests/unit/cli/test_api_delegation.py

```
@@ -51,18 +50,24 @@ def _make_state(with_api_url: bool = False) -> CLIState:
+
+    return ctx
 
 
 # ============================================================================
 # STATUS COMMAND DELEGATION TESTS
 # ============================================================================
 
 
-def test_status_command_delegates_when_api_available(monkeypatch, capsys) -> None:
+def test_status_command_delegates_when_api_available(
+    monkeypatch: MonkeyPatch, capsys: CaptureFixture[str]
```

Ruff (0.14.0)

Unused function argument: `monkeypatch`

(ARG001)